{"Id":91,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:38.4349696Z"},
{"Id":92,"Message":"Entity Framework Core 2.0.1-rtm-125 initialized 'SampleDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None","EventId":10403,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:38.8503732Z"},
{"Id":93,"Message":"Executing action method SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) with arguments ((null)) - ModelState is Valid","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:38.8715943Z"},
{"Id":94,"Message":"GetList()EfService => SampleCoreArch.Entities.Concrete.Category","EventId":1002,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:38.8927554Z"},
{"Id":95,"Message":"Executing ViewResult, running view at path /Views/Home/Index.cshtml.","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.2374433Z"},
{"Id":96,"Message":"Executed action SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) in 2874.1526ms","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.5933973Z"},
{"Id":97,"Message":"Request finished in 3190.1946ms 200 text/html; charset=utf-8","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6158673Z"},
{"Id":98,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/lib/bootstrap/dist/css/bootstrap.css  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6341321Z"},
{"Id":99,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/css/site.css  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6363109Z"},
{"Id":100,"Message":"The file /lib/bootstrap/dist/css/bootstrap.css was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6557289Z"},
{"Id":101,"Message":"The file /css/site.css was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6660625Z"},
{"Id":102,"Message":"Request finished in 40.9394ms 304 text/css","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6752039Z"},
{"Id":103,"Message":"Request finished in 49.082ms 304 text/css","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.6849197Z"},
{"Id":104,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/lib/jquery/dist/jquery.js  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7045068Z"},
{"Id":105,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/images/banner1.svg  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7067911Z"},
{"Id":106,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/images/banner3.svg  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7063313Z"},
{"Id":107,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/lib/bootstrap/dist/js/bootstrap.js  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7114414Z"},
{"Id":108,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/images/banner2.svg  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7135186Z"},
{"Id":109,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/js/site.js?v=ji3-IxbEzYWjzzLCGkF1KDjrT2jLbbrSYXw-AhMPNIA  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.716203Z"},
{"Id":110,"Message":"The file /lib/jquery/dist/jquery.js was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7286534Z"},
{"Id":111,"Message":"The file /images/banner1.svg was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7421025Z"},
{"Id":112,"Message":"The file /images/banner3.svg was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7572749Z"},
{"Id":113,"Message":"The file /lib/bootstrap/dist/js/bootstrap.js was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.769267Z"},
{"Id":114,"Message":"The file /images/banner2.svg was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.782613Z"},
{"Id":115,"Message":"The file /js/site.js was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.7931615Z"},
{"Id":116,"Message":"Request finished in 98.6753ms 304 application/javascript","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.8029219Z"},
{"Id":117,"Message":"Request finished in 109.6841ms 304 image/svg+xml","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.8191499Z"},
{"Id":118,"Message":"Request finished in 131.7985ms 304 image/svg+xml","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.838897Z"},
{"Id":119,"Message":"Request finished in 141.0027ms 304 application/javascript","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.853188Z"},
{"Id":120,"Message":"Request finished in 155.9573ms 304 image/svg+xml","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.8709808Z"},
{"Id":121,"Message":"Request finished in 174.8966ms 304 application/javascript","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.8875489Z"},
{"Id":122,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.8986668Z"},
{"Id":123,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/images/banner4.svg  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.9141855Z"},
{"Id":124,"Message":"The file /lib/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2 was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.9791819Z"},
{"Id":125,"Message":"The file /images/banner4.svg was not modified","EventId":6,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:41.9909547Z"},
{"Id":126,"Message":"Request finished in 105.3871ms 304 font/woff2","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:42.0056329Z"},
{"Id":127,"Message":"Request finished in 130.8658ms 304 image/svg+xml","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:42.048955Z"},
{"Id":128,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/favicon.ico  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:42.0644303Z"},
{"Id":129,"Message":"Sending file. Request path: '/favicon.ico'. Physical path: 'D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\wwwroot\\favicon.ico'","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:42.0860473Z"},
{"Id":130,"Message":"Request finished in 49.3606ms 200 image/x-icon","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:42.1138811Z"},
{"Id":131,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/lib/bootstrap/dist/css/bootstrap.css.map  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:45.7561983Z"},
{"Id":132,"Message":"Sending file. Request path: '/lib/bootstrap/dist/css/bootstrap.css.map'. Physical path: 'D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\wwwroot\\lib\\bootstrap\\dist\\css\\bootstrap.css.map'","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:45.7866481Z"},
{"Id":133,"Message":"Request finished in 65.0789ms 200 text/plain","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:45.820785Z"},
{"Id":134,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:48.9484078Z"},
{"Id":135,"Message":"Entity Framework Core 2.0.1-rtm-125 initialized 'SampleDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None","EventId":10403,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:48.9771501Z"},
{"Id":136,"Message":"Executing action method SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) with arguments ((null)) - ModelState is Valid","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:49.0009309Z"},
{"Id":137,"Message":"GetList()EfService => SampleCoreArch.Entities.Concrete.Category","EventId":1002,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:49.0243474Z"},
{"Id":138,"Message":"Executing ViewResult, running view at path /Views/Home/Index.cshtml.","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:49.0534753Z"},
{"Id":139,"Message":"Executed action SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) in 107.1806ms","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:49.081881Z"},
{"Id":140,"Message":"Request finished in 158.4763ms 200 text/html; charset=utf-8","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:21:49.1079575Z"},
{"Id":141,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.1072345Z"},
{"Id":142,"Message":"Entity Framework Core 2.0.1-rtm-125 initialized 'SampleDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None","EventId":10403,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.1350256Z"},
{"Id":143,"Message":"Executing action method SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) with arguments ((null)) - ModelState is Valid","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.1622386Z"},
{"Id":144,"Message":"GetList()EfService => SampleCoreArch.Entities.Concrete.Category","EventId":1002,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.1869825Z"},
{"Id":145,"Message":"Executing ViewResult, running view at path /Views/Home/Index.cshtml.","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.3363151Z"},
{"Id":146,"Message":"Executed action SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) in 248.5413ms","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.3825034Z"},
{"Id":147,"Message":"Request finished in 308.7487ms 200 text/html; charset=utf-8","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.4189119Z"},
{"Id":148,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/Home/EventLogList  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.5129125Z"},
{"Id":149,"Message":"Entity Framework Core 2.0.1-rtm-125 initialized 'SampleDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None","EventId":10403,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.5459183Z"},
{"Id":150,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/favicon.ico  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.582317Z"},
{"Id":151,"Message":"Executing action method SampleCoreArch.Web.Controllers.HomeController.EventLogList (SampleCoreArch.Web) with arguments ((null)) - ModelState is Valid","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.5823221Z"},
{"Id":152,"Message":"Sending file. Request path: '/favicon.ico'. Physical path: 'D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\wwwroot\\favicon.ico'","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.6154294Z"},
{"Id":153,"Message":"Request finished in 57.2855ms 200 image/x-icon","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:38.6404333Z"},
{"Id":154,"Message":"Executing PartialViewResult, running view at path /Views/Home/EventLogList.cshtml.","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:39.1767946Z"},
{"Id":155,"Message":"Executed action SampleCoreArch.Web.Controllers.HomeController.EventLogList (SampleCoreArch.Web) in 704.9476ms","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:39.24767Z"},
{"Id":156,"Message":"Request finished in 902.7266ms 500 text/html; charset=utf-8","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:23:39.4146982Z"},
{"Id":157,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:10.1871334Z"},
{"Id":158,"Message":"Entity Framework Core 2.0.1-rtm-125 initialized 'SampleDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None","EventId":10403,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:10.583627Z"},
{"Id":159,"Message":"Executing action method SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) with arguments ((null)) - ModelState is Valid","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:10.6098693Z"},
{"Id":160,"Message":"GetList()EfService => SampleCoreArch.Entities.Concrete.Category","EventId":1002,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:10.626974Z"},
{"Id":161,"Message":"Executing ViewResult, running view at path /Views/Home/Index.cshtml.","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.1457937Z"},
{"Id":162,"Message":"Executed action SampleCoreArch.Web.Controllers.HomeController.Index (SampleCoreArch.Web) in 3081.0659ms","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.5220763Z"},
{"Id":163,"Message":"Request finished in 3374.4349ms 200 text/html; charset=utf-8","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.549611Z"},
{"Id":164,"Message":"Request starting HTTP/1.1 GET http://localhost:55095/Home/EventLogList  ","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.6627227Z"},
{"Id":165,"Message":"Entity Framework Core 2.0.1-rtm-125 initialized 'SampleDbContext' using provider 'Microsoft.EntityFrameworkCore.SqlServer' with options: None","EventId":10403,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.6890267Z"},
{"Id":166,"Message":"Executing action method SampleCoreArch.Web.Controllers.HomeController.EventLogList (SampleCoreArch.Web) with arguments ((null)) - ModelState is Valid","EventId":1,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.7108369Z"},
{"Id":167,"Message":"Executed action SampleCoreArch.Web.Controllers.HomeController.EventLogList (SampleCoreArch.Web) in 62.9432ms","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.7464407Z"},
{"Id":168,"Message":"Request finished in 198.4759ms 500 text/html; charset=utf-8","EventId":2,"LogLevel":"Information","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:26:13.8610774Z"},
{"Id":169,"Message":"An unhandled exception has occurred while executing the request\nSystem.ArgumentNullException: Value cannot be null.\r\nParameter name: source\r\n   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\r\n   at SampleCoreArch.Web.Controllers.HomeController.EventLogList() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Controllers\\HomeController.cs:line 38\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:27:47.9464333Z"},
{"Id":170,"Message":"An unhandled exception has occurred while executing the request\nSystem.ArgumentNullException: Value cannot be null.\r\nParameter name: logger\r\n   at Microsoft.Extensions.Logging.LoggerExtensions.LogError(ILogger logger, String message, Object[] args)\r\n   at SampleCoreArch.Core.Service.EntityService`1.GetList(Expression`1 filter) in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Core\\Service\\EntityService.cs:line 69\r\n   at SampleCoreArch.Web.Controllers.HomeController.EventLogList() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Controllers\\HomeController.cs:line 36\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:34:34.6100309Z"},
{"Id":171,"Message":"An unhandled exception has occurred while executing the request\nSystem.ArgumentNullException: Value cannot be null.\r\nParameter name: logger\r\n   at Microsoft.Extensions.Logging.LoggerExtensions.LogError(ILogger logger, String message, Object[] args)\r\n   at SampleCoreArch.Core.Service.EntityService`1.GetList(Expression`1 filter) in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Core\\Service\\EntityService.cs:line 69\r\n   at SampleCoreArch.Web.Controllers.HomeController.EventLogList() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Controllers\\HomeController.cs:line 36\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:36:09.4423533Z"},
{"Id":172,"Message":"An unhandled exception has occurred while executing the request\nSystem.ArgumentNullException: Value cannot be null.\r\nParameter name: logger\r\n   at Microsoft.Extensions.Logging.LoggerExtensions.LogError(ILogger logger, String message, Object[] args)\r\n   at SampleCoreArch.Core.Service.EntityService`1.GetList(Expression`1 filter) in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Core\\Service\\EntityService.cs:line 69\r\n   at SampleCoreArch.Web.Controllers.HomeController.EventLogList() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Controllers\\HomeController.cs:line 36\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:46:24.8937526Z"},
{"Id":173,"Message":"An unhandled exception has occurred while executing the request\nSystem.ArgumentNullException: Value cannot be null.\r\nParameter name: source\r\n   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\r\n   at SampleCoreArch.Web.Controllers.HomeController.EventLogList() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Controllers\\HomeController.cs:line 38\r\n   at lambda_method(Closure , Object , Object[] )\r\n   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionMethodAsync>d__12.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T15:47:33.3679402Z"},
{"Id":174,"Message":"An unhandled exception has occurred while executing the request\nSystem.InvalidOperationException: Templates can be used only with field access, property access, single-dimension array index, or single-parameter custom indexer expressions.\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.ExpressionMetadataProvider.FromLambdaExpression[TModel,TResult](Expression`1 expression, ViewDataDictionary`1 viewData, IModelMetadataProvider metadataProvider)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper`1.GetModelExplorer[TResult](Expression`1 expression)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper`1.DisplayFor[TResult](Expression`1 expression, String templateName, String htmlFieldName, Object additionalViewData)\r\n   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperDisplayExtensions.DisplayFor[TModel,TResult](IHtmlHelper`1 htmlHelper, Expression`1 expression)\r\n   at AspNetCore._Views_Home_EventLogList_cshtml.<ExecuteAsync>d__8.MoveNext() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Views\\Home\\EventLogList.cshtml:line 40\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.PartialViewResult.<ExecuteResultAsync>d__26.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T16:58:17.5418405Z"},
{"Id":175,"Message":"An unhandled exception has occurred while executing the request\nSystem.InvalidOperationException: Templates can be used only with field access, property access, single-dimension array index, or single-parameter custom indexer expressions.\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.ExpressionMetadataProvider.FromLambdaExpression[TModel,TResult](Expression`1 expression, ViewDataDictionary`1 viewData, IModelMetadataProvider metadataProvider)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper`1.GetModelExplorer[TResult](Expression`1 expression)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper`1.DisplayFor[TResult](Expression`1 expression, String templateName, String htmlFieldName, Object additionalViewData)\r\n   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperDisplayExtensions.DisplayFor[TModel,TResult](IHtmlHelper`1 htmlHelper, Expression`1 expression)\r\n   at AspNetCore._Views_Home_EventLogList_cshtml.<ExecuteAsync>d__8.MoveNext() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Views\\Home\\EventLogList.cshtml:line 40\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.PartialViewResult.<ExecuteResultAsync>d__26.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T16:58:26.6364832Z"},
{"Id":176,"Message":"An unhandled exception has occurred while executing the request\nSystem.InvalidOperationException: Templates can be used only with field access, property access, single-dimension array index, or single-parameter custom indexer expressions.\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.Internal.ExpressionMetadataProvider.FromLambdaExpression[TModel,TResult](Expression`1 expression, ViewDataDictionary`1 viewData, IModelMetadataProvider metadataProvider)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper`1.GetModelExplorer[TResult](Expression`1 expression)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper`1.DisplayFor[TResult](Expression`1 expression, String templateName, String htmlFieldName, Object additionalViewData)\r\n   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperDisplayExtensions.DisplayFor[TModel,TResult](IHtmlHelper`1 htmlHelper, Expression`1 expression)\r\n   at AspNetCore._Views_Home_EventLogList_cshtml.<ExecuteAsync>d__8.MoveNext() in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Views\\Home\\EventLogList.cshtml:line 40\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()\r\n   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.PartialViewResult.<ExecuteResultAsync>d__26.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.102","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-02T17:00:25.9711606Z"},
